### PR TITLE
Cleaning up config, removing outdated CSS

### PR DESCRIPTION
**Pull request by madrilene**
## Updates

- Have corrected wrong references to custom properties.
- Cleaned up the config file.
- Removed CSS from the webc components (is not scoped in the `njk` context - I may find another solution for this in the future, but I don't think it's bad if there is some unused CSS in the global.css file).

## Questions or Recommendations

- About the `inlineMarkdown` hack where you want to remove the surrounding `<p>` tags - I'm not quite sure why we need that? Could it be that this is a leftover from another strategy? You use it in `article.njk` for example like this:
  ```html
  <h1>{{ title | inlineMarkdown | safe }}</h1>
  ```
And I'm not quite sure why?

- How do you want to handle mobile menu behaviour? Wrap pills / drawer menu.
- I plan to add a `postcss` plugin for color-mix that converts the values into hex colors, for unsupported browsers. Ok with that?
- If we only use simple shapes like circles and rectangles, I would possibly use CSS instead of SVG for the so-called "decorative" elements.

### CODE REVIEW

**Comment by tobie** [comment]
> About the inlineMarkdown hack where you want to remove the surrounding <p> tags - I'm not quite sure why we need that? Could it be that this is a leftover from another strategy? You use it in article.njk for example like this:
<h1>{{ title | inlineMarkdown | safe }}</h1>
And I'm not quite sure why?

To allow for markdown in titles. 

> How do you want to handle mobile menu behaviour? Wrap pills / drawer menu.

let’s discuss in a dedicated issue. 

> I plan to add a postcss plugin for color-mix that converts the values into hex colors, for unsupported browsers. Ok with that?

I’m entirely fine having a very degraded experience in older browsers as long as the content remains readable. I wouldn’t worry about these. 

> If we only use simple shapes like circles and rectangles, I would possibly use CSS instead of SVG for the so-called "decorative" elements.

svg’s fine.

File: src/_eleventy/utils.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import {default as createImgAttr} from "./utils/image-attribution.js";
```
Comment (tobie) [inline src/_eleventy/utils.js:1]: Let's rename `createImgAttr` to something more clear while we're at it. No idea what I was thinking.

Comment (tobie) [reply src/_eleventy/utils.js:1]: e.g. `image-metadata-index.js` and `createImageMetadataIndex`

Not great, but at least explicit.

Also, no need for the "{ default as alias }" syntax. What I had initially just works.

Comment (madrilene) [reply src/_eleventy/utils.js:1]: You're right! Not sure why I did that.

**Comment by madrilene** [comment]
I have also made the "sun" change we discussed, both renaming it and assigning it to the speaking contents.

**Pull request merged**